frontend/src/scenes/error-tracking/ErrorTrackingScene.tsx

```diff
@@ -32,6 +33,10 @@ export function ErrorTrackingScene(): JSX.Element {
                 width: '50%',
                 render: CustomGroupTitleColumn,
             },
+            "toJSONString(arrayReduce('sumMap', arrayMap(x -> map(x,1), groupArray(toHour(timestamp))))) -- Sparkline":
```

Comment:
@pauldambra mainly created this to get feedback from you on the best way to generate this data in a ClickHouse query.

Basically here:
- `groupArray` gives us an array of every hour an event happened in e.g. `[0,23,16,12,12,12,12,13,13]`
- Uses `arrayMap` to create a map of counts from each value  e.g. `[{0: 1},{23: 1},{16: 1}, {12: 1}{12: 1}{12: 1}{12: 1}{13: 1}{13: 1}]`
- `arrayReduce('sumMap', ...)` can then combine all those maps together to give e.g. `{0: 1, 23:1, 16:1, 12:3, 13: 2}`

There's still work to be done on the frontend then because the ideal structure is an array of counts zero padded for missing hours (done in `SparklineColumn`).

Perhaps I'm vastly overcomplicating this but I'm maybe not familiar enough with the ClickHouse functions to know better

Reply:
The other option is to return the results of the `groupArray` but ideally we'd aggregate in CH because that array could be quite large for very frequent exceptions

Reply:
big +1 to doing as much as possible in CH
particularly if you end up with large datasets it's soooo much cheaper than looping and summing in python

Reply:
Fly-by note: I would call this column not for the visualization type ("Sparkline"), but for the data represented (e.g. "Volume by hour")

Reply:
Ended up with a nicer implementation in https://github.com/PostHog/posthog/commit/7d7784251adbd3077cfbb66826de2d6e6cd95f7f

Good call on the naming... updated